sunbeam-python/sunbeam/plugins/validation/plugin.py

```diff
@@ -0,0 +1,473 @@
+        """Disable OpenStack Integration Test Suite (tempest)."""
+        super().disable_plugin()
+
+    @click.command()
+    @click.argument("options", nargs=-1)
+    def configure_validation(self, options: Optional[List[str]] = None) -> None:
+        """Configure validation plugin.
+
+        Run without arguments to view available configuration options.
+
+        Run with key=value args to set configuration values.
+        For example: sunbeam configure validation schedule="*/30 * * * *"
+        """
+        if not self._configure_preflight_check():
+            raise click.ClickException(
+                "'observability' plugin is required for configuring validation plugin."
+            )
+
+        if not options:
+            console.print(
+                "Config options available: \n\n"
+                "schedule: set a cron schedule for running periodic tests.  "
+                "Empty disables.\n\n"
+                "Run with key=value args to set configuration values.\n"
+                'For example: sunbeam configure validation schedule="*/30 * * * *"'
+            )
+            return
+
+        config_changes = validated_config_args(parse_config_args(options))
+
+        if config_changes.schedule is not None:
+            jhelper = JujuHelper(self.client, self.snap.paths.user_data)
+            with console.status("Configuring validation plugin ..."):
+                run_sync(
+                    jhelper.set_application_config(
+                        OPENSTACK_MODEL,
+                        TEMPEST_APP_NAME,
+                        config={"schedule": config_changes.schedule},
+                    )
+                )
+                console.print(f"Schedule has been set to '{config_changes.schedule}'")
+
+    @click.command()
+    @click.option(
+        "-r",
+        "--regex",
+        default="",
+        help=(
+            "A list of regexes, whitespace separated, used to select tests from"
+            " the list."
+        ),
+    )
+    @click.option(
+        "-e",
+        "--exclude-regex",
+        default="",
+        help="A single regex to exclude tests.",
+    )
+    @click.option(
+        "-t",
+        "--serial",
+        is_flag=True,
+        default=False,
+        help="Run tests serially. By default, tests run in parallel.",
+    )
+    @click.option(
+        "-l",
+        "--test-list",
+        default="",
+        help=(
+            "Use a predefined test list. See `sunbeam validation test-lists`"
+            " for available test lists."
+        ),
+    )
+    @click.option(
+        "-o",
+        "--output",
+        type=click.Path(),
+        default=None,
+        help=(
+            "Download the full log to output file. "
+            "If not provided, the output can be retrieved later "
+            "by running `sunbeam validation get-last-result`."
+        ),
+    )
+    def run_validate_action(
+        self,
+        smoke: bool = False,
```

Comment:
Since we no longer have the `smoke` option, this needs to be removed.

Reply:
Right, thanks!